### PR TITLE
Relaxing the SHACL to allow separate schema:Organization items in graph

### PR DESCRIPTION
**Pull request by coret**
Fix https://github.com/netwerk-digitaal-erfgoed/dataset-register/issues/265

### CODE REVIEW

File: test/validator.test.ts [hunk 1]
```diff
@@ -21,6 +21,13 @@ describe('Validator', () => {
     expect(report.state).toEqual('valid');
   });
 
+  it('accepts minimal valid Schema.org dataset', async () => {
```
Comment (ddeboer) [inline test/validator.test.ts:24]: Description of the test case doesn’t match the contents.

File: test/datasets/dataset-schema-org-valid-plus-organization.jsonld [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+         },
+         {
+            "@type": "DataDownload",
+            "encodingFormat": "application/ld+json",
+            "contentUrl": "http://data.bibliotheken.nl/id/dataset/rise-alba.jsonld"
+         }
+      ]
+   },
+   {
+      "@context": "http://schema.org",
```
Comment (ddeboer) [inline test/datasets/dataset-schema-org-valid-plus-organization.jsonld:39]: `http` versus `https` is inconsistent.

**Review by ddeboer — CHANGES_REQUESTED** [review]
Do we also have a test that ensures that the dataset’s creator is still validated?

File: shacl/register.ttl [hunk 3]
```diff
@@ -77,7 +77,6 @@ reg:DistributionShape
 
 reg:OrganizationShape
     a sh:NodeShape ;
-    sh:targetClass schema:Organization ;
```
Comment (ddeboer) [inline shacl/register.ttl:80]: Or there other shapes that we should remove the `targetClass` from?

File: test/datasets/dataset-schema-org-valid-plus-organization.jsonld (hunk 2 above)
Comment (coret) [reply test/datasets/dataset-schema-org-valid-plus-organization.jsonld]: ok

File: shacl/register.ttl (hunk 3 above)
Comment (coret) [reply shacl/register.ttl:80]: yes, dcat:OrganizationShape
what's left (on purpose): schema:DataCatalog, schema:Dataset, schema:DataDownload and dcat:Dataset, dcat:Distribution

**Review by ddeboer — APPROVED** [review]

**Pull request merged**